.github/workflows/publish-template.yml

```diff
@@ -18,10 +18,7 @@ jobs:
       - name: install rust
         uses: dtolnay/rust-toolchain@stable
 
-      - name: compile publish binary
-        run: rustc scripts/publish.rs --crate-type bin --out-dir scripts
-
       - name: publish to crates.io
-        run: ./scripts/publish ${{ inputs.crate }}
+        run: cargo-xtask publish ${{ inputs.crate }}
```

Comment:
is `cargo-xtask` supposed to be `cargo xtask` ?

Reply:
Oops - yes it is :)